shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt

```diff
@@ -83,19 +87,25 @@ interface ScreenshotTest {
         val heightInDp = heightInPx ?: height
         runOnUi {
             ViewHelpers.setupView(view)
-                    .setExactHeightPx(heightInDp)
-                    .setExactWidthPx(width)
-                    .layout()
+                .setExactHeightPx(heightInDp)
```

Comment:
Not related with this PR but I think you don't need the `heightInDp` variable (also, it's misleading)

Reply:
Actually that variable is not needed! Thanks 😃